src/test/java/org/jnode/fs/xfs/XfsFileSystemTest.java

```diff
@@ -173,6 +182,128 @@ private static String getDate(long date) {
         return simpleDateFormat.format(new java.util.Date(date));
     }
 
+    @Test
+    public void testShortFormAttribute() throws Exception {
+        File testFile = FileSystemTestUtils.getTestFile("org/jnode/fs/xfs/extended_attr.img");
+        try (FileDevice device = new FileDevice(testFile, "r")) {
+            XfsFileSystemType type = new XfsFileSystemType();
+            XfsFileSystem fs = type.create(device, true);
+            final INode shortAttributeINode = fs.getINode(11075L);
+            final XfsEntry entry = new XfsEntry(shortAttributeINode, "", 0, fs, null);
+            assertThat(shortAttributeINode.getAttributesFormat(),is(1L));// Short form attribute format
+            final List<FSAttribute> attributes = entry.getAttributes();
+            assertThat(attributes,hasSize(1));
+            final FSAttribute attribute = attributes.get(0);
+            assertThat(attribute.getName(),is("selinux"));
+            assertThat(attribute.getValue(),is("unconfined_u:object_r:unlabeled_t:s0"));
```

Comment:
This unit test fails now that the return value from `getValue()` is a byte array.